bin/plugin/commands/changelog.js

```diff
@@ -162,7 +142,14 @@ function getIssueFocus( issue ) {
  * @return {string} The formatted changelog string.
  */
 function formatChangelog( milestone, pullRequests ) {
-	let changelog = '= ' + milestone.replace( 'PL Plugin ', '' ) + ' =\n\n';
+	const version = milestone.match( /\d+\.\d+(\.\d+)?(-[A-Za-z0-9\.]+)?$/ );
```

Comment:
No need to escape `.` in a character range. In PhpStorm, this results in a warning:

> Redundant character escape '\.' in RegExp 

So we can remove it:

```suggestion
	const version = milestone.match( /\d+\.\d+(\.\d+)?(-[A-Za-z0-9.]+)?$/ );
```